0006/README.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 0006
+title: Resiliency in Tinkerbell Provisioner.
+status: Ideation
+authors: Chitrabasu Khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
+This RFD proposes to have a resilient framework for Tinkerbell Provisioner.
+
+
+## Goals:
+
+- Bring resiliency in Tinkerbell. 
+- Tinkerbell as a setup should be highly available.
+
+## Content: 
+
+- We would like to have a framework where Tinkerbell as a solution is fault-tolerant.
+- All most all of the core services are managed by provisioner. So, it will be good to have a framework where provisioner is highly available.
```

Comment:
Can you be more precise? Why it is not "reliable" today?

Reply:
@gianarb I have updated the doc on the reliability part. I am proposing this solution which should be indepented on the form of deployment or setup type.  

@detiber yes, K8s can be one of the ways to do it. With K8s we can have multiple instances of the services plus scalability can also be managed by K8s itself. 
One challenge we could possibly face is (which this PR is also trying to address is) what is the master K8 node goes down 9due to network or hardware failure) in that case also Tinkerbell as setup will fail. Though, We can have multi-master K8s cluster to solve that problem.